service/internal/auth/authn.go

```diff
@@ -415,7 +417,7 @@ func GetRawAccessTokenFromContext(ctx context.Context, l *logger.Logger) string
 	return ""
 }
 
-func validateDPoP(accessToken jwt.Token, acessTokenRaw string, dpopInfo receiverInfo, headers []string, logger *logger.Logger) (jwk.Key, error) {
+func (a Authentication) validateDPoP(accessToken jwt.Token, acessTokenRaw string, dpopInfo receiverInfo, headers []string) (jwk.Key, error) {
```

Comment:
nit: sp -- "acessTokenRaw" -> "accessTokenRaw" but looks like this is separate from your changes so feel free to ignore for now

Reply:
I think there has been a move to add a spellchecker to our linter, but we haven't turned it on